model_compiler/tox.ini

```diff
@@ -10,7 +10,7 @@ commands = {envpython} tools/install-tensorrt.py
            mypy --install-types --non-interactive -p model_compiler -p tests.model_compiler
            pylint src tests
 
-           cpu: pytest -m 'not gpu_test' --cov --cov-config .coveragerc-cpu -n auto
+           cpu: pytest -m 'not gpu_test and not dtu_test' --cov --cov-config .coveragerc-cpu -n auto
```

Comment:
Maybe we should add another test suit named dtu?

Reply:
There is no good solution here for the time being, other places have been changed. thank you

Reply:
@shimei0203  I mean that we should add another test suit as cpu and nvidia gpu, for example: 
```
cpu: pytest ...
gpu: pytest ...
dtu: pytest ...
```
The reason for this is that gpu test suits can not run on dtu hardware, so I think we should distinguish the tests between gpu and dtu.